Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.17.2
+FROM alpine:3.17.3
```

Comment:
Should we change this to a SHA for increased security?

Reply:
I would be fine with that. So far we don't do it anywhere I think, so 🤷